proxy/http/HttpSM.h

```diff
@@ -519,6 +522,18 @@ class HttpSM : public Continuation, public PluginUserArgs<TS_USER_ARGS_TXN>
   /// Update the milestones to track time spent in the plugin API.
   void milestone_update_api_time();
 
+  sockaddr *
```

Comment:
Pay attention to existing style - note none of the other methods are defined inline.